src/core.js

```diff
@@ -841,8 +841,9 @@
 
   };
 
-  win.xtag = xtag;
-  if (typeof define == 'function' && define.amd) define(xtag);
+  if (typeof define === 'function' && define.amd) define(xtag);
```

Comment:
because if I'm doing CommonJS w/ Browserify, I don't want you polluting a global, brah